### PR TITLE
More checkboxes in the PR

### PR DESCRIPTION
**Pull request by zippy1981**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes #<enter issue number>)
 - [ ] New feature (non-breaking change, adds functionality)
 - [ ] Breaking change (effects multiple commands or functionality)
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Pester test is included
 - [x] **Add items to this checklist that would be relevant to doc and C# Changes**

### Purpose
check items for documentation and C# related PRs.

### CODE REVIEW

**Comment by wsmelton** [comment]
Can you expand these more possibly to explain what they are for? If they are all solely on the C# side of the project you might want to just make a new section in the template for it, and expand on what details you may need to review such changes.

> Adding code coverage to existing functionality

What do you mean by "code coverage"? Is this part of Pester code or something?

> Nunit test is included

Is this specific to C# ?

> Documentation

Is this help syntax in each command or some part of C# project?

> Build system

I'm assuming this is dealing with the library compile process?

**Comment by zippy1981** [comment]
@wsmelton 

[code coverage](https://en.wikipedia.org/wiki/Code_coverage) means adding unit tests that test things not currently tested.  

Nunit is the C# unit testing framework. 

Documenation would be any documetation, the C#, the powershell help comments, the readme etc. 

Build system would be appvoyer stuff, stuff that happens to turn the C# code into the DLL, and any other "yak shaving" like if we added some kind of linter to do static analysis. 

Basically areas I think we lack checkboxes for:

*  C# changes
* A PR to improve appvoyer stuff
* Addin unit tests to existing code
* a pull request that is pure technical writing (and thanks to @alevyinroc for doing the lords work here).

**Comment by wsmelton** [comment]
I've been wanting to redo that list because I've noticed on average no one but those directly involved in the project tend to use it. I tried to not be too excessive with a checklist for the PR template or even the issue template because of this very reason. We could re-work the current checklist so we try to keep it to 5-7 options, I would basically put them as what you just commented on:

So go from this:
 - [ ] Bug fix (non-breaking change, fixes #<enter issue number>)
 - [ ] New feature (non-breaking change, adds functionality)
 - [ ] Breaking change (affects multiple commands or functionality)
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Pester test is included

To this:
 - [ ] Bug fix ( fixes #<issue number> )
 - [ ] New feature ( adds functionlity, enhancement )
 - [ ] C# changes ( adjustments to project/library )
 - [ ] Adjustment to Appveyor process
 - [ ] Adding/Updating Pester test
 - [ ] Technical writing updates (Documentation)

Terms like 'build system' or 'nunit' are only going to be understood by a handful of folks that have the C# knowledge or direct knowledge of our project setup.

**Comment by potatoqualitee** [comment]
Thanks, shawn!

**Comment by potatoqualitee** [comment]
Oh oops! LOL, this was Zippy! Thanks @zippy1981 !
